com.unity.ml-agents/Runtime/Sensors/RayPerceptionSensorComponentBase.cs

```diff
@@ -252,16 +252,26 @@ internal void UpdateSensor()
             }
         }
 
+        internal int SensorObservationAge()
+        {
+            if (m_RaySensor != null)
+            {
+                return Time.frameCount - m_RaySensor.DebugLastFrameCount;
+            }
+
+            return 0;
+        }
+
         void OnDrawGizmosSelected()
         {
-            if (m_RaySensor?.debugDisplayInfo?.rayInfos != null)
+            if (m_RaySensor?.m_rayPerceptionOutput?.RayOutputs != null)
```

Comment:
```suggestion
            if (m_RaySensor?.RayPerceptionOutput?.RayOutputs != null)
```

Reply:
Thanks, I'll make `m_{r,R}ayPerceptionOutput` private (and fix the capitalization)